graphql-schemas/schema.yml

```diff
@@ -80,6 +81,11 @@ confs:
   - { name: maxRetries, type: int }
   - { name: mailDomain, type: string }
 
+- name: LdapGroupsSettings_v1
+  fields:
+  - { name: contactList, type: string, isRequired: true }
+  - { name: credentials, type: VaultSecret_v1, isRequired: true }
```

Comment:
These 2 fields are required but in json schema they are not required

Reply:
🤦 fixed in 6d46ca8cc719edd